assignment1.txt

```diff
@@ -0,0 +1,11 @@
+Yael Kelmer 
+
+1.The significance of return 0; is to return the value of  0 to let the computer know that the function has run completely. However, at this point in           technology, compilers are advanced enough to send this 0 to the computer without the user typing return 0;. For this reason, it is not really necessary to put  return 0;, but it is a good habit to have. 
+
+2.The code contains "%d", but the variable that is defined is a float. The      command "%d" calls on a variable that is a digit (integer), which the float is  not (41.5). In order to call the right variable, the code would have to say "%f" instead of "%d".
```

Comment:
Correct, but you're missing a couple of errors! check to see if you can find them!